gamification/src/main/webapp/WEB-INF/application-context.xml

```diff
@@ -12,7 +12,7 @@
     <context:annotation-config/>
     <context:component-scan base-package="juja.microservices.gamification"/>
     <mongo:repositories
-            base-package="juja.microservices.gamification.model.repository"/>
```

Comment:
@sLisnychyi let's use annotated configuration instead of xml. Please, refactore this point